### PR TITLE
add dependabot configuration

### PR DESCRIPTION
**Pull request by yeikel**
See https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (yeikel) [inline .github/dependabot.yml:6]: If needed, we can change this to weekly

**Comment by tsegismont** [comment]
Thank you but I don't believe this is necessary. GitHub provides notifications about releases that we use to determine if a new version is required.

**Comment by yeikel** [comment]
> Thank you but I don't believe this is necessary. GitHub provides notifications about releases that we use to determine if a new version is required.

Are these pull requests created automatically?

Because notifications are easy to miss and it's not only about security alerts

**Comment by tsegismont** [comment]
No they are not created automatically.

**Pull request closed without merging**